### PR TITLE
Add jobs of openshift origin + public clouds

### PR DESCRIPTION
**Pull request by liu-sheng**
This change add jobs:
  -  openshift-origin-v3.9.0-functional-test-huaweicloud
  - openshift-origin-v3.9.0-functional-test-opentelekomcloud
  - openshift-origin-v3.9.0-functional-test-telefonica
  - openshift-origin-v3.9.0-functional-test-orange

Implement: #theopenlab/openlab#149

### CODE REVIEW

**Comment by liu-sheng** [comment]
I will restart zuul when no jobs running, that may fix the above error.

**Comment by liu-sheng** [comment]
recheck

File: zuul.d/jobs.yaml [hunk 1]
```diff
@@ -1039,3 +1039,56 @@
     timeout: 18000
     secrets:
       - fusioncloud_credentials
+
+- job:
+    name: openshift-origin-functional-test-huaweicloud
+    parent: init-test
+    description: |
+      Run functional tests of openshift-origin repo of v3.9.0 release against huaweicloud
+    run: playbooks/openshift-orgin-functional-test-public-clouds/run.yaml
```
[the remark this answers is not in the document]
Comment (liu-sheng) [reply zuul.d/jobs.yaml]: done

File: zuul.d/jobs.yaml [hunk 2]
```diff
@@ -1039,3 +1039,56 @@
...
+    secrets:
+      - huaweicloud_credentials
+
+- job:
+    name: openshift-origin-functional-test-opentelekomcloud
+    parent: init-test
+    description: |
+      Run functional tests of openshift-origin repo of v3.9.0 release against opentelekomcloud
+    run: playbooks/openshift-orgin-functional-test-public-clouds/run.yaml
+    post-run: playbooks/openshift-orgin-functional-test-public-clouds/post.yaml
```
[the remark this answers is not in the document]
Comment (liu-sheng) [reply zuul.d/jobs.yaml]: done

File: zuul.d/jobs.yaml [hunk 3]
```diff
@@ -1039,3 +1039,56 @@
...
+      inventory_name: telefonica
+    secrets:
+      - telefonica_credentials
+
+- job:
+    name: openshift-origin-functional-test-orange
+    parent: init-test
+    description: |
+      Run functional tests of openshift-origin repo of v3.9.0 release against orange cloud
+    run: playbooks/openshift-orgin-functional-test-public-clouds/run.yaml
```
[the remark this answers is not in the document]
Comment (liu-sheng) [reply zuul.d/jobs.yaml]: done

**Comment by liu-sheng** [comment]
@kiwik now, the every openshift cluster deployment will use 2 VMs with 2 floating ips, and also the test case of loadbalancer will also need to create a loadbalancer with a floating ip.

yes, I agree to add another periodic pipeline, now we put too many jobs together in 1 periodic pipeline.

**Comment by kiwik** [comment]
> @kiwik now, the every openshift cluster deployment will use 2 VMs with 2 floating ips, and also the test case of loadbalancer will also need to create a loadbalancer with a floating ip.
> 
> yes, I agree to add another periodic pipeline, now we put too many jobs together in 1 periodic pipeline.

2 vm and 2 floating ips looks not too many, let us merge this PR to check when it's running.

**Pull request merged**